### PR TITLE
LRQA-60634 | master

### PR DESCRIPTION
**Pull request by zhangyan0701**
https://issues.liferay.com/browse/LRQA-60634

### CODE REVIEW

**Comment by zhangyan0701** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase [hunk 1]
```diff
@@ -135,4 +135,89 @@ definition {
...
+
+		FileInstall.deleteWarOffline(liferayDependencyWars = "test-theme.war");
+
+		Theme.deployThemeWar(themeName = "${appName}");
+
+		Navigator.openSitePage(
+			pageName = "Theme Test Page",
+			siteName = "Guest");
+
+		SikuliAssertElementNotPresent(locator1 = "Theme#TEST_THEME_7_3_HOME_PAGE_PNG");
```
Comment (john-co) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase:205]: It's weird to me that there's no alert message or other cue to let the user know that this theme has been blacklisted if they try to deploy theme again. I imagine the user being confused if no console logs that give positive or negative feedback. Can you create a bug ticket for this if that's the case?

**Comment by john-co** [comment]
looks good to me!

**Comment by john-co** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase (hunk 1 above)
Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase:205]: cc/ @zhangyan0701

Comment (zhangyan0701) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase:205]: Sorry for delay, looks like I missed your comment. 

There is a console message when redeploy theme, but is same as when uninstall it.
`Stopping blacklisted bundle test-theme`
So this message appears twice in console in this case.

Comment (john-co) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase:205]: I see. So no way to truly be able to assert the 2nd `Stopping blacklisted bundle test-theme`...but the follow up assertions should be good enough.

**Pull request closed without merging**